pkg/util/prefix_wildcard_test.go

```diff
@@ -0,0 +1,45 @@
+package util
+
+import "testing"
+
+func TestMatches(t *testing.T) {
+	tcs := []struct {
+		name      string
+		pw        PrefixWildcard
+		candidate string
+		matches   bool
+	}{
+		{
+			name:      "exact text match",
+			pw:        PrefixWildcard("kube-system"),
+			candidate: "kube-system",
+			matches:   true,
+		},
+		{
+			name:      "no glob, wrong text",
```

Comment:
Add a no glob, only prefix test, we would expect no match.

Reply:
Good call.  Done.